data.go

```diff
@@ -15,6 +15,8 @@ import (
 	"strings"
 )
 
+const DefaultMaxFormSize = 2048
```

Comment:
I like for all exported identifiers to have doc comments. Mind adding a comment here? It doesn't have to be long. Just something like:

```
// DefaultMaxFormSize is the default maximum form size (in bytes) used by the Parse function.
```

Also, I know this is my fault, but I think 2048 is probably much too small. I want to find a value that is likely to be safe (i.e. is unlikely to cause OOM exceptions in most environments) and allows for common things like text files, pdfs, and smaller images. Shall we try something around 500 KB?